packages/svelte/src/internal/client/proxy/proxy.js

```diff
@@ -100,12 +100,12 @@ function unwrap(value, already_unwrapped = new Map()) {
 }
 
 /**
- * @template {StateObject} T
+ * @template T
  * @param {T} value
  * @returns {Record<string | symbol, any>}
```

Comment:
The return type needs to be adjusted, too
```suggestion
 * @template T
 * @param {T} value
 * @returns {T}
```

Reply:
That's what I had originally, I'll just go back to that.